### DISCUSSION TITLE
check spelling spanish not working?

### DISCUSSION THREAD

**Discussion by 9acca9** [vkbo/novelWriter#1372] [Help] [answered]
the spell check is not working for me, i cant select, if i try, nothing change:
![Captura desde 2023-02-21 01-21-46](https://user-images.githubusercontent.com/25393371/220246973-70125535-7e8b-4bd7-b23a-c8c7980fb3c5.png)

And if i go to preferences i see like this:
![Captura desde 2023-02-21 01-18-50](https://user-images.githubusercontent.com/25393371/220247047-6f739e1a-2326-4807-b298-397b584de286.png)

What i can do?

**Comment by vkbo** [comment, marked as answer]
You don't have any dictionaries installed that the Enchant library can find. What operating system are you using?

**Reply by 9acca9** [reply]
Im using Manjaro GNOME version.

**Reply by vkbo** [reply]
Are you running novelWriter from AppImage or from source?

As long as enchant is installed and has access to your local dictionaries, it should pick up any hunspell or aspell dictionaries you've installed on your system.

**Reply by 9acca9** [reply]
From source. I cant see now the computer, but when I go back I will see.

Edit: sorry about the thread thing.

**Reply by vkbo** [reply]
Ok. If you're running from source, make sure Enchant is installed and properly set up. If novelWriter cannot contact Enchant properly, it will instead set up a mock spell check interface that will have no dictionaries.

If you start novelWriter from command line with the `--debug` flag, there should be a line near the top saying:

```
[2023-02-21 19:15:01,394]          config.py:771   DEBUG     Checking package 'pyenchant': OK
```

If pyenchant was loaded correctly.

No problem with the thread. It just gets tricky to follow when the thread is split. The user interface is not well designed here.

**Reply by 9acca9** [reply]
And here is the problem:
```
[2023-02-22 16:58:07,990]          config.py:771   DEBUG     Checking package 'pyenchant': OK
[2023-02-22 16:58:08,320]      spellcheck.py:81    WARNING   Enchant found no dictionary for language ''

```

And the solution:
`sudo pacman -S aspell-es `

Thanks for the help!
